### ISSUE TITLE
After installing this addon, ember serve is failing

### ISSUE THREAD

**Issue by rishimukherjee** [shipshapecode/ember-flatpickr#41]
Here is the error I am receiving.

```
Could not start watchman; falling back to NodeWatcher for file system events.
Visit http://ember-cli.com/user-guide/#watchman for more info.
Cannot use 'in' operator to search for 'enabled' in undefined
TypeError: Cannot use 'in' operator to search for 'enabled' in undefined
    at normalizeModuleOptions (/Users/rishimukherjee/sezzle/merchant-dashboard/node_modules/ember-flatpickr/node_modules/ember-cli-node-assets/index.js:83:20)
    at /Users/rishimukherjee/sezzle/merchant-dashboard/node_modules/ember-flatpickr/node_modules/ember-cli-node-assets/index.js:72:16
    at /Users/rishimukherjee/sezzle/merchant-dashboard/node_modules/ember-flatpickr/node_modules/ember-cli-node-assets/index.js:52:42
    at Array.map (native)
    at CoreObject.getModuleOptions (/Users/rishimukherjee/sezzle/merchant-dashboard/node_modules/ember-flatpickr/node_modules/ember-cli-node-assets/index.js:51:38)
    at CoreObject.doImports (/Users/rishimukherjee/sezzle/merchant-dashboard/node_modules/ember-flatpickr/node_modules/ember-cli-node-assets/index.js:29:10)
    at CoreObject.included (/Users/rishimukherjee/sezzle/merchant-dashboard/node_modules/ember-flatpickr/node_modules/ember-cli-node-assets/index.js:22:10)
    at CoreObject.superWrapper [as included] (/Users/rishimukherjee/sezzle/merchant-dashboard/node_modules/core-object/lib/assign-properties.js:32:18)
    at /Users/rishimukherjee/sezzle/merchant-dashboard/node_modules/ember-cli/lib/models/addon.js:258:34
    at Array.map (native)
    at CoreObject.eachAddonInvoke (/Users/rishimukherjee/sezzle/merchant-dashboard/node_modules/ember-cli/lib/models/addon.js:256:24)
    at CoreObject.included (/Users/rishimukherjee/sezzle/merchant-dashboard/node_modules/ember-cli/lib/models/addon.js:433:10)
    at CoreObject.superWrapper [as included] (/Users/rishimukherjee/sezzle/merchant-dashboard/node_modules/core-object/lib/assign-properties.js:32:18)
    at CoreObject.included (/Users/rishimukherjee/sezzle/merchant-dashboard/node_modules/ember-flatpickr/index.js:15:26)
    at CoreObject.superWrapper [as included] (/Users/rishimukherjee/sezzle/merchant-dashboard/node_modules/core-object/lib/assign-properties.js:32:18)
    at EmberApp.<anonymous> (/Users/rishimukherjee/sezzle/merchant-dashboard/node_modules/ember-cli/lib/broccoli/ember-app.js:468:15)
    at Array.filter (native)
    at EmberApp._notifyAddonIncluded (/Users/rishimukherjee/sezzle/merchant-dashboard/node_modules/ember-cli/lib/broccoli/ember-app.js:463:45)
    at new EmberApp (/Users/rishimukherjee/sezzle/merchant-dashboard/node_modules/ember-cli/lib/broccoli/ember-app.js:138:8)
    at module.exports (/Users/rishimukherjee/sezzle/merchant-dashboard/ember-cli-build.js:6:13)
    at FastBootBuild.toTree (/Users/rishimukherjee/sezzle/merchant-dashboard/node_modules/ember-cli-fastboot/lib/broccoli/fastboot-build.js:73:41)
    at CoreObject.buildFastBootTree (/Users/rishimukherjee/sezzle/merchant-dashboard/node_modules/ember-cli-fastboot/index.js:139:26)
```

**Comment by rwwagner90** [comment]
@rishimukherjee I see some stuff about fastboot in that stack trace. Are you running `ember serve` here or using fastboot?

**Comment by rwwagner90** [comment]
@rishimukherjee you still having problems?

**Comment by rwwagner90** [comment]
@rishimukherjee are you there? I would love to help you figure this out.

**Comment by rishimukherjee** [comment]
Hi. I didn't get any email. So missed it. Really sorry.

I didn't try after that we moved to using simple masks for date input. But yes, I use `ember serve` to run the project and my version of Ember is `2.7.0`. My node version is `3.10.9`.

**Comment by rwwagner90** [comment]
@rishimukherjee no problem. Are you sure that is your node version? I think node went from 0.12 to 4. Can you please tell me what `node --version` outputs?

**Comment by rwwagner90** [comment]
@rishimukherjee the stack trace also seems to indicate the use of fastboot, even though you say you are just running `ember serve`.

There was an issue with fastboot, which I just fixed, and released version 1.0.2. So please try it and see if the problem is gone.

**Comment by rwwagner90** [comment]
Since I have not heard from you, I am going to close this. Please reach out if you are still having issues!
